go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/tikv/client-go/v2
 
-go 1.18
+go 1.19
```

Comment:
Can we keep it unchanged at least for the root `go.mod`? Now, the minimal supported go version for this library is still 1.18.